easybuild/easyconfigs/s/Spyder/Spyder-5.0.5-foss-2020b.eb

```diff
@@ -0,0 +1,278 @@
+    }),
+    ('isort', '5.9.2', {
+        'checksums': ['f65ce5bd4cbc6abdfbe29afc2f0245538ab358c14590912df638033f157d555e'],
+    }),
+    ('pylint', '2.9.5', {
+        'checksums': ['1f333dc72ef7f5ea166b3230936ebcfb1f3b722e76c980cb9fe6b9f95e8d3172'],
+    }),
+    ('ujson', '4.0.2', {
+        'checksums': ['c615a9e9e378a7383b756b7e7a73c38b22aeb8967a8bfbffd4741f7ffd043c4d'],
+    }),
+    ('python-language-server', '0.36.2', {
+        'modulename': 'pyls',
+        'checksums': ['9984c84a67ee2c5102c8e703215f407fcfa5e62b0ae86c9572d0ada8c4b417b0'],
+    }),
+    ('pyrsistent', '0.18.0', {
+        'checksums': ['773c781216f8c2900b42a7b638d5b517bb134ae1acbebe4d1e8f1f41ea60eb4b'],
+    }),
+    ('python-jsonrpc-server', '0.4.0', {
+        'modulename': 'pyls_jsonrpc',
+        'checksums': ['62c543e541f101ec5b57dc654efc212d2c2e3ea47ff6f54b2e7dcb36ecf20595'],
+    }),
+    ('python-slugify', '5.0.2', {
+        'modulename': 'slugify',
+        'checksums': ['f13383a0b9fcbe649a1892b9c8eb4f8eab1d6d84b84bb7a624317afa98159cab'],
+    }),
+    ('pyxdg', '0.27', {
+        'modulename': 'xdg',
+        'checksums': ['80bd93aae5ed82435f20462ea0208fb198d8eec262e831ee06ce9ddb6b91c5a5'],
+    }),
+    ('rope', '0.19.0', {
+        'checksums': ['64e6d747532e1f5c8009ec5aae3e5523a5bcedf516f39a750d57d8ed749d90da'],
+    }),
+    ('QDarkStyle', '3.0.2', {
+        'checksums': ['55d149cf5f40ee297397f1818e091118cefb855a4a9c5c38566c47acd2d8c7ae'],
+    }),
+    ('QtAwesome', '1.0.3', {
+        'checksums': ['d37bbeb69ddc591e5ff036b741bda8d1d92133811f1f5a7150021506f70b8e6e'],
+    }),
+    ('qtconsole', '5.1.1', {
+        'checksums': ['bbc34bca14f65535afcb401bc74b752bac955e5313001ba640383f7e5857dc49'],
+    }),
+    ('QtPy', '1.9.0', {
+        'checksums': ['2db72c44b55d0fe1407be8fba35c838ad0d6d3bb81f23007886dc1fc0f459c8d'],
+    }),
+    ('SecretStorage', '3.3.1', {
+        'modulename': 'secretstorage',
+        'checksums': ['fd666c51a6bf200643495a04abb261f83229dcb6fd8472ec393df7ffc8b6f195'],
+    }),
+    ('three-merge', '0.1.1', {
+        'checksums': ['60f6afe144595560d63ae32625351bcef3b94733b54eb97800a9feb0f3d9d970'],
+    }),
+    ('parso', '0.7.0', {
+        'checksums': ['908e9fae2144a076d72ae4e25539143d40b8e3eafbaeae03c1bfe226f4cdf12c'],
+    }),
+    ('Rtree', '0.9.7', {
+        'checksums': ['be8772ca34699a9ad3fb4cfe2cfb6629854e453c10b3328039301bbfc128ca3e'],
+    }),
+    ('spyder', version, {
+        'preinstallopts': """sed -i "s/'pyqt5.*,//g" setup.py && sed -i "s/'pyqtwebengine.*,//g" setup.py && """,
+        'checksums': ['b1b5c06b5cd9902f9746a083fd760bb4de3fa4129bc8c1c1ef2bcd84002bc45f'],
+    }),
+    ('spyder-kernels', '2.0.5', {
+        'modulename': 'spyder_kernels',
+        'checksums': ['4428a9aa8f94987f366fad3afda427933a131cca5fd9c361c8553c564eae011e'],
+    }),
+    ('text-unidecode', '1.3', {
+        'checksums': ['bad6603bb14d279193107714b288be206cac565dfa49aa5b105294dd5c4aab93'],
+    }),
+    ('traitlets', version, {
```

Comment:
Already included in deps (slightly different version, but maybe still compatible). I also think the "version" here would be wrong regardless